packages/firestore/src/remote/remote_store.ts

```diff
@@ -243,16 +243,15 @@ export class RemoteStore {
    */
   disableNetwork(): Promise<void> {
     // Set the OnlineState to failed so get()'s return from cache, etc.
-    return this.disableNetworkInternal(OnlineState.Failed);
+    this.disableNetworkInternal(OnlineState.Failed);
+    return Promise.resolve();
   }
 
   /**
    * Disables the network, setting the OnlineState to the specified
    * targetOnlineState.
    */
-  private disableNetworkInternal(
-    targetOnlineState: OnlineState
-  ): Promise<void> {
+  private disableNetworkInternal(targetOnlineState: OnlineState) {
```

Comment:
Can you explicitly declare the return type to be void now?

Reply:
Done.